tests/cli/test_correct_scraper.py

```diff
@@ -15,8 +15,8 @@
 
 tests = {
     'release': {
-        'args': ['-v', '23.0.1'],
-        'fname': 'firefox-23.0.1.en-US.linux64.tar.bz2',
+        'args': ['-v', '23.0.1', '-p', 'win32'],
```

Comment:
Without the platform specified it would fail on other platforms than linux. To be consistent I used win32 everywhere for now. With mock we can enhance this later.